docker/Dockerfile

```diff
@@ -49,7 +49,7 @@ RUN LLVM_VERSION=7.0.1 && \
     SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
     wget --quiet ${LLVM_URL} --output-document=${LLVM_ARCHIVE} && \
     wget --quiet ${LLVM_URL}.sig --output-document=${LLVM_ARCHIVE}.sig && \
-    gpg --keyserver pool.sks-keyservers.net --recv-keys ${LLVM_KEY} && \
+    gpg --keyserver keys.gnupg.net --recv-keys ${LLVM_KEY} && \
```

Comment:
What was the rational for changing?  I don't have a strong opinion on this, I am just curious what make you think that this will prevent the occasional failure to reach the server to occur.

Reply:
It was more than an occasional failure recently. We were constantly not able to fetch keys from sks-servers, pretty much at every PR. There is no reason to suggest that gnupg will be more robust, but thought I'd give it a try and see how it behaves. So far, so good.